api/tokens.go

```diff
@@ -148,3 +151,72 @@ func (api *API) RefreshHandler(ctx context.Context, w http.ResponseWriter, req *
 
 	return models.NewSuccessResponse(jsonResponse, http.StatusCreated, headers), nil
 }
+
+//SignOutAllUsersHandler bulk refresh token invalidation for panic sign out handling
+func (api *API) SignOutAllUsersHandler(ctx context.Context, w http.ResponseWriter, req *http.Request) (*models.SuccessResponse, *models.ErrorResponse) {
+	usersList := models.UsersList{}
+	listUserInput := usersList.BuildListUserRequest("status=\"Enabled\"", "username", int64(0), &api.UserPoolId)
+	listUserResp, err := api.CognitoClient.ListUsers(listUserInput)
+	if err != nil {
+		return nil, models.NewErrorResponse(http.StatusInternalServerError, nil, err)
```

Comment:
Add a log error event before returning here - it gives us some context what the app was trying to do before it failed.